suites/nautilus/ansible/tier_0_deploy_rpm_ceph.yaml

```diff
@@ -0,0 +1,64 @@
+tests:
```

Comment:
Do we need a separate RPM/baremetal script? Shall we incorporate this scenario with the other core test?